api/graphql/schema.graphql

```diff
@@ -796,10 +796,17 @@ enum CandidateExpiryFilter {
   ALL
 }
 
+enum CandidateSuspendedFilter {
+  ACTIVE
+  SUSPENDED
+  ALL
+}
+
 input UserPoolFilterInput {
   poolId: ID!
   expiryStatus: CandidateExpiryFilter = ACTIVE
   statuses: [PoolCandidateStatus!]
+  suspendedStatus: CandidateSuspendedFilter = ALL
```

Comment:
I think this should be set to ACTIVE by default. We won't usually care about suspended candidates.

Reply:
oops, you are right https://github.com/GCTC-NTGC/gc-digital-talent/pull/5918/commits/fefb4a73624a021b24fce5f071c738d754e3107a